patches/minecraft/net/minecraft/client/renderer/block/model/ItemTransform.java.patch

```diff
@@ -1,14 +1,38 @@
 --- a/net/minecraft/client/renderer/block/model/ItemTransform.java
 +++ b/net/minecraft/client/renderer/block/model/ItemTransform.java
-@@ -14,7 +_,11 @@
- import net.minecraftforge.api.distmarker.Dist;
- import net.minecraftforge.api.distmarker.OnlyIn;
- 
-+/**
-+ * @deprecated use {@link com.mojang.math.Transformation} through {@link net.minecraftforge.client.extensions.IForgeBakedModel#handlePerspective}
-+ */
- @OnlyIn(Dist.CLIENT)
-+@Deprecated
- public class ItemTransform {
-    public static final ItemTransform f_111754_ = new ItemTransform(new Vector3f(), new Vector3f(), new Vector3f(1.0F, 1.0F, 1.0F));
+@@ -20,11 +_,17 @@
     public final Vector3f f_111755_;
+    public final Vector3f f_111756_;
+    public final Vector3f f_111757_;
++   public final Vector3f rightRotation;
+ 
+    public ItemTransform(Vector3f p_111760_, Vector3f p_111761_, Vector3f p_111762_) {
```

Comment:
A second rotation is completely optional and most people have no need for it. Since the default value for it is pre-populated with a null rotation if the original constructor is used, I think it'd be fine to leave it as-is.